src/routes.tsx

```diff
@@ -36,269 +38,331 @@ enum EmbarkStory {
   SwedenQuoteCartNeeder = 'Web Onboarding SE - Quote Cart Needer',
 }
 
-export interface ServerSideRoute {
-  path: string | RegExp
+export type ServerSideRoute = {
   titleTextKey: string
   metaDescriptionTextKey?: string
   ogImage?: string
   status?: number
 }
 
-export const serverSideRoutes: ServerSideRoute[] = [
-  {
-    path: localePathPattern + '/forever/:code?',
-    titleTextKey: 'FOREVER_LANDINGPAGE_TITLE',
-    metaDescriptionTextKey: 'FOREVER_LANDINGPAGE_DESCRIPTION',
-    ogImage:
-      'https://www.hedvig.com/new-member-assets/social/forever-notifications.jpg',
-  },
-  {
-    path: localePathPattern + '/new-member/download',
-    titleTextKey: 'DOWNLOAD_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/connect-payment',
-    titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/connect-payment/direct',
-    titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/connect-payment/success',
-    titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/connect-payment/fail',
-    titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/connect-payment/retry',
-    titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/(offer|sign)',
-    titleTextKey: 'OFFER_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/sign/success',
-    titleTextKey: '',
-  },
-  {
-    path: localePathPattern + '/new-member/sign/fail',
-    titleTextKey: '',
-  },
-  {
-    path: localePathPattern + '/new-member/(offer|sign)/:id',
-    titleTextKey: 'OFFER_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/new-member/:name?/:id?',
-    titleTextKey: 'START_PAGE_TITLE',
-  },
-  {
-    path: localePathPattern + '/login',
-    titleTextKey: 'Text',
-  },
-  {
-    path: /^(?!\/?new-member-assets).*$/,
-    titleTextKey: 'FOUR_OH_FOUR_PAGE_TITLE',
-    status: 404,
-  },
-]
-
-interface ReactPageRoute {
-  path: string
+type ClientSideRoute = {
   Component?: React.ComponentType<any>
   render?: (props: RouteComponentProps<any>) => React.ReactNode
-  exact?: boolean
+  exact: boolean
+  isHiddenInProd?: boolean
+}
+
+export type Route = {
+  path: string | RegExp
+  serverRouteData?: ServerSideRoute
+  clientRouteData?: ClientSideRoute
 }
 
-export const reactPageRoutes: ReactPageRoute[] = [
+export const landingRoute = '/new-member'
```

Comment:
Does this need to be exported?

Reply:
Doesn't seem like it. If I remember it correctly I had some idea of using that in the million places where we hard code `/new-member` throughout the code base, but probably then thought that would mean an unreasonably big PR 🙃